### ISSUE TITLE
Řazení výsledků vyhledávání

### ISSUE THREAD

**Issue by zabak** [moravianlibrary/kramerius-for-ios#132]
Výsledky by mělo jít řadit minimálně podle roku, názvu a autora a možná i podle druhu dokumentu - vzestupně i sestupně

**Comment by honza-rychtar** [comment]
Jak chces radit podle autora?
Pres autora by se melo spis filtrovat - ve fasetech by byli nejcasteji se vyskytujici autori a ty by mohlo byt mozne vybirat.
Pokud radit podle autora, tak ne dokumenty, ale samotny seznam autoru, ktery by mohl byt k dispozici.
Podle druhu dokumentu nedava razeni vybec smysl - jak to chces udelat? Podle abecedy? Nemame ani ustalene nazvoslovi - monografie/kniha, periodikum/noviny, ... a to ani nemluvim o prekladech do jinych jazyku.
U druhu dokumentu ma zase smysl spis filtrovat.

Razeni podle nepovinnych poli je problem - co kdyz neni pole vyplnene? Zaznam je pak az uplne na konci.
Coz je hlavne problem u razeni podle roku. Kdyz seradis vsechny vysledky podle roku, tak ti to veci bez vyplneneho roku hodi az na konec a uzivatel ma dojem, ze tam ty zaznamy vubec nejsou ...
Razeni podle roku je taky problem z duvodu zapisu - neni to vzdy rok, ale v pripade periodik spis rozsah let - takze ciselne to neseradis. A i kdyby to slo, tak jak bys to radil? Podle zacatku rozsahu roku, podle konce, podle stredu, ...

**Comment by zabak** [comment]
Jo, máš pravdu, jednodušší bude doplnit filtry a ošetří to většinu věcí.
